sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -439,6 +447,7 @@ protected void initializeDeploymentPlanManager() {
             deployPlan = deploy.get();
         }
         deploymentPlanManager = new DefaultPlanManager(deployPlan);
+        deploymentPlanManager.getPlan().getStrategy().proceed();
```

Comment:
Guessing this is a fix for when `CanaryStrategy` is used? Maybe add a comment explaining the logic here (cross-reference with `DefaultPlanManager`'s interrupt() call?).